docs/reference/setup/install/targz-start.asciidoc

```diff
@@ -60,6 +40,12 @@ that supports arrays and assume that Bash is available at `/bin/bash`.
 As such, Bash should be available at this path either directly or via a
 symbolic link.
 
+[discrete]
+==== Enroll nodes in an existing cluster
+
 To enroll additional nodes in your cluster, create an enrollment token with the
-`elasticsearch-create-enrollment-token` tool. You can then start a node with the `--enrollment-token` parameter so that it
-{ref}/configuring-stack-security.html#stack-enroll-nodes[joins an existing cluster].
\ No newline at end of file
+`elasticsearch-create-enrollment-token` tool. You can then start a node with the `--enrollment-token` parameter so that it joins an existing cluster.
+
+// The following include pulls in steps for enrolling nodes in a cluster from
+// a security page in the x-pack folder
+include::../../../../x-pack/docs/en/security/configuring-stack-security.asciidoc[tag=enroll-nodes]
```

Comment:
This include is valid for linux and osx but not on windows as the CLI is elasticsearch-create-enrollment-token.bat in windows and we need backslashes in windows too

Reply:
Also I think we can rework the wording to make it a little more obvious what action needs to be run in which node? We use "a node", "other node", "the node" and I think it might get confusing for someone who reads this for the first time. 

Could there be a way/word to clearly and consistently through our docs refer to nodes and make the distinvtion between an existing node ( that can generate enrollment tokens ) and a new node ( That will use enrollment tokens to enroll to an existing cluster ) ?

Reply:
I moved the steps for enrolling nodes to their own file, added variables, and then used `include::` entries to pull in the information. I also clarified "new" vs. "existing" nodes.